src/LndClient/Data/SendCoins.hs

```diff
@@ -13,7 +13,8 @@ import qualified Proto.Lightning_Fields as LnGRPC
 
 data SendCoinsRequest = SendCoinsRequest
   { addr :: Text,
-    amount :: MSat
+    amount :: MSat,
+    sendAll :: Bool
```

Comment:
This flag looks a bit scary because of boolean blindness and because one small bool field completely change behaviour. Let's have two data constructors `SendCoinsRequest {} | SendCoinsAll Addr`